src/css/components/loading.scss

```diff
@@ -2,25 +2,66 @@
 $z-foreground: 200;
 
 .loading {
-  height: 50%;
+  bottom: 0;
+  height: 100%;
   left: 0;
-  position: relative;
+  position: absolute;
+  right: 0;
   top: 0;
   width: 100%;
   z-index: $z-foreground;
 }
 
+.loading-container {
+  display: block;
+  height: $grid-9;
+  position: relative;
+  width: 100%;
+
+  & + .loading-container {
+    display: none;
+  }
+}
+
 .loading-relative {
   top: auto;
 }
 
-$indicator-size: 3.5rem;
+.loading-information {
```

Comment:
the new general container for the block loading indicator — sized so it can accommodate text